WalletWasabi/Discoverability/NostrCoordinatorPublisher.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using NNostr.Client;
+using WalletWasabi.Bases;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Discoverability;
+
+public class NostrCoordinatorPublisher : PeriodicRunner
+{
+	public NostrCoordinatorPublisher(TimeSpan period, AnnouncerConfig config, Network network) : base(period)
+	{
+		Key = ECPrivKey.Create(config.KeyBytes);
+		CoordinatorConfiguration = new NostrCoordinatorConfiguration(config.CoordinatorDescription, new Uri(config.CoordinatorUri), network);
```

Comment:
This CoordinatorConfiguarion property is not necessary. It is useless. It is here just to make more complex the creation of the event. Remove it please.